src/moj/components/button-menu/_button-menu.scss

```diff
@@ -1,95 +1,45 @@
-/* ==========================================================================
-   #BUTTON GROUP
-   ========================================================================== */
+$moj-button-hover-colour: #767676;
+$moj-datepicker-mid-grey: #949494;
```

Comment:
This is deliberately badly named in order to highlight the need for refactoring!